test/test_freezegun.py

```diff
@@ -1,36 +1,39 @@
-import pytest
-import time
 import datetime
+import time
 
+import pytest
 from freezegun import freeze_time
+
 from libfaketime import fake_time
 
 # TODO
 # - Fix time.localtime
 # - Fix time.strftime
-# - Fix python2 time.time with 1970-01-01
+
+assert time
+assert datetime
```

Comment:
what are these asserts for?

Reply:
This is a little hack so the linter won't remove the imports, as it believes the module are not called in the file.
Actually there is a `eval` that does call them dynamically, so the asserts make the linter happy.